scopes.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Access scopes
+---
+
+diaspora* API features access scopes (or just 'scopes') in order to implement resource access management for API clients.
+
+Scopes are part of [OAuth 2.0 specification][oauth2] which is a base for [OpenID Connect Core 1.0][connect] implemented by diaspora*.
+
+When the API client requests an access token it sends the list of scopes it wants to get access to accoring to the selected authorization flow (see ["Authentication"][authentication]).
+
+Approved scopes may differ from what was requested by the client.
+
+[OpenID Connect Core 1.0][connect] defines a few scopes which are specific for getting the authorized user information. API client must include the `openid` scope to the scope request list. `openid` scope means "allow OpenID Connect authentication" and if missing then API authentication will fail.
```

Comment:
"API clients"
"The `openid` scope gives access to using OpenID Connect authentication and if missing the API authentication will fail."